samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -962,10 +962,14 @@ std::map<std::string, ov::TensorVector> get_tensors_static_case(const std::vecto
     return blobs;
 }
 
-void copy_tensor_data(ov::Tensor& dst, const ov::Tensor& src) {
-    if (src.get_shape() != dst.get_shape() || src.get_byte_size() != dst.get_byte_size()) {
-        throw std::runtime_error(
-            "Source and destination tensors shapes and byte sizes are expected to be equal for data copying.");
+void copy_tensor_data(ov::Tensor& dst, const ov::Tensor& src, bool vpu) {
```

Comment:
Why do we need this flag?!

Reply:
To apply this WA for vpu only.

Reply:
OpenVINO is generic software where all plugins behave in a similar way (the only exception is properties which can be plugin specific or optional functionality)
It means such kind of WAs are not acceptable by definition.

Reply:
I see. Thanks!